### PR TITLE
Update Vonage Video iOS SDK to 2.27.2 and Android SDK to 2.27.1

### PR DESCRIPTION
**Pull request by jeffswartz**
### Contributing checklist
- [x] Code must follow existing styling conventions
- [x] Added a descriptive commit message
- [ ] Sample apps updated if needed

### CODE REVIEW

**Review by v-kpheng — APPROVED** [review]
LGTM! 💪 🚀

**Review by jintgeorge — COMMENTED** [review]
@jeffswartz Left a comment.

File: opentok-react-native.podspec [hunk 1]
```diff
@@ -17,5 +17,5 @@ Pod::Spec.new do |s|
   s.source_files  = "ios/**/*.{h,m,swift}"
 
   s.dependency 'React'
-  s.dependency 'OTXCFramework','2.27.0'
+  s.dependency 'OTXCFramework','2.27.1'
```
Comment (pvenum) [inline opentok-react-native.podspec:20]: This is 2.27.2

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+# 2.27.3  (March 2024)
+
+- [Update]: This version updates the Vonage Video iOS SDK version to 2.27.2 and the Vonage Video Android SDK version to 2.27.1.
+
+*Note:* In Android 6.0 (`API Level 23`) and higher, the OpenTok React Native SDK automatically adds the camera access permission. However, an app or user can disable it independently of the SDK.
```
Comment (jeffswartz) [inline CHANGELOG.md:5]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
...
+
```
Comment (jeffswartz) [inline CHANGELOG.md:6]:
```suggestion
```

**Review by jintgeorge — APPROVED** [review]
LGTM

**Pull request merged**